test/ui-2/test/DropDownButtonStep.test.js

```diff
@@ -0,0 +1,35 @@
+const timeout = process.env.SLOWMO ? 30000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+  path = fs.realpathSync('file://../examples/index.html');
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+describe('add step dropdown', ()=>{
```

Comment:
Let's make this look professional. Could you please capitalize the test name too?